### DISCUSSION TITLE
The current tabcontrol is difficult to use

### DISCUSSION THREAD

**Discussion by kaisame** [canton7/Stylet#359] [Q&A]
The current tabcontrol is difficult to use,For example, it is troublesome to customize each tabitem
`
<TabControl>
            <TabItem>
                <TabItem.Header>
                    <Grid />
                </TabItem.Header>
            </TabItem>
            <TabItem>
                <TabItem.Header>
                    <Grid />
                </TabItem.Header>
            </TabItem>
            <TabItem>
                <TabItem.Header>
                    <Grid />
                </TabItem.Header>
            </TabItem>
  </TabControl>
`

I want to do something in the grid,It is difficult to use stylet tabcontrol

**Comment by canton7** [comment]
I'm not sure what you're asking for I'm afraid.

Stylet doesn't provide a TabControl: that's part of WPF. I can't change how WPF's TabControl works.

Stylet does provide a nice way to have a View and ViewModel for each tab in a TabControl, [see the docs](https://github.com/canton7/Stylet/wiki/Screens-and-Conductors#conductortcollectiononeactive). But it doesn't seem that you're using this from the small amount of code you posted.
